### PR TITLE
Clean up: viewer specification

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions 
- establish alphabetical order
- establish correct order within a property
- fix indentation

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

File: schemas/miscellaneous/viewerSpecification.schema.tpl.json [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+    },
+    "cameraPosition": {
+      "_instruction": "Enter the camera position that a viewer should use.",
+      "_embeddedTypes": [
+        "https://openminds.ebrains.eu/sands/CoordinatePoint"
+      ]
+    },
+    "preferredDisplayColor": {  
+      "_instruction": "Add the preferred color that a viewer should display.",
+      "_linkedTypes": [
```
Comment (lzehl) [inline schemas/miscellaneous/viewerSpecification.schema.tpl.json]: should we make this embedded ? colors can of course be reused but from a query point of you this additional connection to other annotations through color is not really useful. What do you think?

**Review by lzehl — COMMENTED** [review]
@UlrikeS91 @Peyman-N @apdavison @aeidi89 please see my comment. I generally approve this PR.

File: schemas/miscellaneous/viewerSpecification.schema.tpl.json (hunk 1 above)
Comment (lzehl) [reply schemas/miscellaneous/viewerSpecification.schema.tpl.json]: change ColorMap to controlledTerms

Comment (lzehl) [reply schemas/miscellaneous/viewerSpecification.schema.tpl.json]: keep linkedType then

**Review by lzehl — COMMENTED** [review]
@apdavison @Peyman-N @aeidi89 I've changed the linkage to the colors now to a _linkedCategories (color). I'll do the related PRs immediately. Please verify and merge if you are happy

**Review by Peyman-N — APPROVED** [review]

**Pull request merged**